tests/testthat/test-get_lifetable.R

```diff
@@ -0,0 +1,3 @@
+test_that("object returns", {
+  expect_type(get_lifetable(), "list")
```

Comment:
As these files are large I think we might want to avoid including the download in the testing (it'll really slow down the dev cycle having to do this every time).

As in other comments, if you can break down the larger wrapper functions into the smallest minimum units and create small, quick tests for those functions that might work better.

If helpful you could include some small "dummy" version on the downloaded data in the testing folder to test all the post-processing functions on